addons/music/fnc_playMusic.sqf

```diff
@@ -1,48 +1,53 @@
 /* ----------------------------------------------------------------------------
 Function: CBA_fnc_playMusic
+
 Description:
     A function used to play music. 
+    
 Parameters:
     1: CLASS Class to play
     2: INT Position to play from (default: 0)
     3: BOOL allowed to interupt already playing music (default: true)
+    
+Returns:
+    BOOL -true if started playing music
+    
 Example:
     (begin example)
-    _bool = ["LeadTrack01_F_Bootcamp", 30, true] call CBA_fnc_playMusic;
+        _bool = ["LeadTrack01_F_Bootcamp", 30, true] call CBA_fnc_playMusic;
     (end example)
-Returns:
-    BOOL
+    
+
 Author:
     Fishy
 ---------------------------------------------------------------------------- */
 #include "script_component.hpp"
-private ["_config","_data","_canPlay","_duration","_return"];
-params [["_className",""],["_time",'name'],["_overWrite", true]];
-if (_className == "") exitWith {
+
+params [["_className","",[""]],["_time",0,[0]],["_overWrite", true,[true]]];
+
+if (_className isEqualTo "") exitWith {
     WARNING("No class given"); 
     false
 };
 
+If ((!_overWrite)&&{[true] call CBA_fnc_getMusicPlaying}) exitWith {false};
```

Comment:
I don't think `!_overWrite` needs parentheses but doesn't matter probably. 
A few more spaces to both sides of the `&&` would make this a little more readable